routers/usersRouter.js

```diff
@@ -0,0 +1,171 @@
+const usersDb = require("../data/helpers/usersDb");
+const bcrypt = require("bcryptjs");
+const jwt = require("jsonwebtoken");
+
+const express = require("express");
+const router = express.Router();
+const secret = "She-Ra";
+
```

Comment:
You've got JWTs down Angela. This file is really the core of this module. Creating a token, verifying it, attaching it to request/endpoints, and securing routes by requiring the token 👍